Index: main/service/codici_catastali_service.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import csv\r\n# questa parte e' da inserire in un service apposito.\r\n\r\n\r\nclass CodiciCatastali:\r\n    comune, regione, provincia, codici_catastale = [], [], [], []\r\n    with open('C:/Users/franc/PycharmProjects/pythonProject/HelloWorld!!/data/codici.csv', 'r') as file_in:\r\n        reader = csv.reader(file_in, delimiter=',')\r\n        for row in reader:\r\n            comune.append(row[1])\r\n            regione.append(row[2])\r\n            provincia.append(row[3])\r\n            codici_catastale.append(row[4])\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main/service/codici_catastali_service.py b/main/service/codici_catastali_service.py
--- a/main/service/codici_catastali_service.py	
+++ b/main/service/codici_catastali_service.py	
@@ -1,5 +1,4 @@
 import csv
-# questa parte e' da inserire in un service apposito.
 
 
 class CodiciCatastali:
